homeassistant/components/fritzbox/config_flow.py

```diff
@@ -36,30 +41,31 @@
 RESULT_SUCCESS = "success"
 
 
-class FritzboxConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+class FritzboxConfigFlow(ConfigFlow, domain=DOMAIN):
     """Handle a AVM FRITZ!SmartHome config flow."""
 
     VERSION = 1
 
-    def __init__(self):
+    def __init__(self) -> None:
         """Initialize flow."""
-        self._entry = None
-        self._host = None
-        self._name = None
-        self._password = None
-        self._username = None
+        self._entry: ConfigEntry | None = None
+        self._host: str | None = None
+        self._name: str = ""
```

Comment:
That's not what I meant. Argument in `_get_entry` is needed to pass `name` which can't be `None` at that point as regular `str`.
```suggestion
        self._name: str | None = None
```